AssemblerLib/LocalToGlobalIndexMap.h

```diff
@@ -31,32 +35,31 @@ namespace AssemblerLib
 class LocalToGlobalIndexMap
 {
 public:
+#ifdef USE_PETSC
+    typedef MathLib::RowColumnIndices<PetscInt> RowColumnIndices;
```

Comment:
@wenqing How is the PetscInt different from maximum unsigned integer?
Actually this code should not be dependent on PETSc but on MeshLib::NodePartitionMesh or (base) MeshLib::Mesh.
This part handles indices of some meshes and parts of them and is not related to PETSc or any other type of parallelization.